.travis.yml

```diff
@@ -149,6 +149,11 @@ stages:
 branches:
   only: [master]
 
+# MANUALLY ADDED. Do not remove.
```

Comment:
Does mono_repo not yet support adding `after_success` commands?

Reply:
No, it does not.